code/modules/paperwork/pen.dm

```diff
@@ -93,17 +93,58 @@
 	icon_state = "fancypen"
 
 /obj/item/pen/multi/gold
-	name = "Gilded Pen"
+	name = "gilded pen"
 	desc = "A golden pen that is gilded with a meager amount of gold material. The word 'Nanotrasen' is etched on the clip of the pen."
 	icon_state = "goldpen"
 	pen_colour_shift = 0
 
 /obj/item/pen/multi/fountain
-	name = "Engraved Fountain Pen"
+	name = "engraved fountain pen"
 	desc = "An expensive looking pen."
```

Comment:
this could _maybe_ do with a better description

Reply:
It does look expensive though

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/25437893/b08877e0-c322-4b67-af25-da463cd595b1)
 i don't really know how else to expand this really it's not exactly a specific NTR item since currently it still spawns in captain office + ERTs get it

Reply:
that looks good, yeah